lib/models/speaker.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:ecellapp/core/res/strings.dart';
+import 'package:equatable/equatable.dart';
+
+class Speaker extends Equatable {
+  final String comapny;
+  final String contact;
+  final String createdAt;
+  final String description;
+  final String email;
+  final int experience;
+  final bool flag;
+  final int id;
+  final String modifiedAt;
+  final String name;
+  final String profilePic;
```

Comment:
add doc comments on non-obvious variables as to what kind of value they hold

Reply:
From the result of a conversation with the backend team, I will be removing unnecessary fields that also include but not limited to email and contact.